targets/go/source/PlayFabHttp.go.ejs

```diff
@@ -0,0 +1,165 @@
+package playfab
+
+// This code was generated by a tool. Any changes may be overwritten
+
+import (
+	"bytes"
+	"compress/gzip"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"reflect"
+	"strings"
+	"time"
+)
+
+// Settings includes various conf options for PlayFab HTTP call
+type Settings struct {
+	VersionString            string
+	RequestGetParams         map[string]string
+	ProductionEnvironmentURL string
+	EnableCompression        bool
+	VerticalName             string
+	TitleId                  string
+	DisableSsl               bool
+	APIPath                  string
+}
+
+// NewSettings returns a pointer to a Settings struct that carries default values
+func NewSettings() *Settings {
+	return &Settings{
+		VersionString:            "GoSdk-",
+		RequestGetParams:         map[string]string{"sdk": ""},
+		ProductionEnvironmentURL: ".playfabapi.com", // This is only for customers running a private cluster.  Generally you shouldn't touch this
+		EnableCompression:        false,
+		VerticalName:             "", // The name of a customer vertical. This is only for customers running a private cluster. Generally you shouldn't touch this
+		TitleId:                  "",
+		DisableSsl:               false,
+	}
+}
+
+// GetFullURL returns the full URL to be used for the API call
+func (p *Settings) GetFullURL(apiPath string) string {
+	var output string
+
+	if !strings.HasPrefix(p.ProductionEnvironmentURL, "http") {
+		if p.VerticalName != "" {
+			output = "https://" + p.VerticalName
+		} else {
+			output = "https://" + p.TitleId
+		}
+	}
+
+	output += p.ProductionEnvironmentURL
+	output += apiPath
+
+	firstParam := true
+	for key, val := range p.RequestGetParams {
+		if firstParam {
+			output += "?"
+			firstParam = false
+		} else {
+			output += "&"
+		}
+
+		output += fmt.Sprintf("%s=%s", key, val)
+	}
+
+	return output
+}
+
+// Request executes the actual HTTP call
+func Request(p *Settings, b []byte, apiPath string, authKey string, authValue string) (map[string]interface{}, error) {
+
+	client := &http.Client{
+		Timeout: time.Second * 30, // 30 seconds timeout should be enough
+	}
+
+	if p.DisableSsl {
+		client.Transport = &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
This is a potential security risk,  I pinged you on slack about it.. Is there a better solution in GO where we can validate the certificate?

Reply:
Go does TLS validation by default on HTTP requests.
I copied this from the PHP SDK, I’d be more than happy to remove it if you think it’s unnecessary.